pages/article.js

```diff
@@ -1,28 +1,49 @@
 import { Component } from 'react'
+import { Query } from 'react-apollo'
+import gql from 'graphql-tag'
+import withData from '../lib/withData'
+
 import Layout from '../components/Layout'
-import UserComments from '../components/UserComments'
+import Article from '../components/Article'
+
+class Articles extends Component {
+  constructor (props) {
+    super(props)
+
+    this.articleQuery = gql`
+      query article($slug: String!) {
+        article(slug:$slug) {
+          author
+          title
+          text
+          createdAt
+          image
+          comments {
+            body
+            authorName
+            CreatedAt
```

Comment:
should be lower case `c`. You should be able to confirm these queries by testing it through graphiql or insomnia